tests/integration.spec.ts

```diff
@@ -92,7 +92,7 @@ test('integration testing', async (t) => {
    * Close Client & Server
    */
   await new Promise(resolve => setImmediate(resolve))
-  eventStream.cancel()
+  setImmediate(() => eventStream.cancel())
```

Comment:
Thanks for the PR!

Cloud you please explain whether it's necessary to change `.cacel()` to `setImmediate(.cacel())` when we have already a `await new Promise(resolve => setImmediate(resolve))` above?

I think we have already achieved the same goal with the `await new Promise(resolve => setImmediate(resolve))` because it has waited for the event loop to be cleared before we call `.cacel()`,

Reply:
@huan Yeah, you are correct. Both will work fine.